bundles/org.openhab.binding.ism8/src/main/java/org/openhab/binding/ism8/server/KnxNetFrame.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ism8.server;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link KnxNetFrame} is used for handling the received KNX.Net frames
+ *
+ * @author Hans-Reiner Hoffmann - Initial contribution
+ */
+@NonNullByDefault
+public class KnxNetFrame {
+    public static byte[] knxHeader = new byte[6];
+    public static byte[] connectionHeader = new byte[4];
+
+    private static final Logger logger = LoggerFactory.getLogger(KnxNetFrame.class);
```

Comment:
Upper case

Reply:
Updated. To maintain the readability I was applying `_` (e.g. `KNX_HEADER`).